src/app/components/ccd-search/ccd-search.component.ts

```diff
@@ -87,7 +100,8 @@ export class CcdSearchComponent implements OnInit {
             }
             // tslint:disable-next-line:max-line-length
```

Comment:
If not required we can remove commented codes.

Reply:
That is the Eslint rule format. This is not commented line.